pyvo/registry/regtap.py

```diff
@@ -715,8 +727,9 @@ def access_modes(self):
 
     def get_interface(self,
                       service_type: str,
-                      lax: bool = True,
-                      std_only: bool = False):
+                      lax: bool = False,
+                      std_only: bool = False, *,
```

Comment:
`*` will need to come earlier, I think before `lax`, but in v1.6, so drop the `*` for now (I really don't think it's a good thing to have functions that take a bunch of bools as parameters without explicitly spelling out the parameters)